kalite/distributed/static/js/distributed/exercises/perseus-helpers.js

```diff
@@ -99,25 +113,17 @@ Exercises.PerseusBridge = {
     render_item: function(item_data) {
 
         Exercises.PerseusBridge.itemMountNode = document.createElement("div");
-        var itemRenderer = Exercises.PerseusBridge.itemRenderer = Perseus.ItemRenderer({
+
```

Comment:
These changes are pretty much copied verbatim from the Perseus repo 'test-render.html'.